bin/runner.go

```diff
@@ -41,6 +41,7 @@ func main() {
 
 		if err := experiment.SetENV(engineDetails, clients); err != nil {
 			klog.V(0).Infof("Unable to patch ENV due to error: %v", err)
+			recorder.ExperimentSkipped(engineDetails.Experiments[i])
```

Comment:
After adding this event, will it also add the reason for skipping? It would be great if you add the event log in the PR